### PR TITLE
[App] Main Page Layout 

### PR DESCRIPTION
**Pull request by aniku777tw**
與這支 PR 有關聯的 Asana 卡片:

URL: no link.

# Description

Main Page Layout

# Step by Step

1. New animation for  switching pages .
2. Changed some colors make it looks more pro.
3. Took a cup of tea.
4. Add the order cards list (It is scrollable ).
5. When the card clicked , it will show up a simple dialog.
6. Clean some code make it reusable.
7. Have fun

# Screenshot or more (Optional)
1.Main page.
image
2.When the card clicked , it will show up a simple dialog.
image

-----------------

提交 PR 前，請先檢查以下是否已完成：

* [O ] 該分支是從最新 `master` 切出來 (如果不是，請先 rebase/merge 它)
* [X ] 是否新增測試，或被現有的測試覆蓋
* [O ] 這支 PR 是否有明確的標題、內容描述

-----------------

補充說明:

### CODE REVIEW

File: client/lib/pages/block_goods_page.dart [hunk 1]
```diff
@@ -11,3 +11,7 @@ class BlockGoodsPage extends StatelessWidget {
     ));
   }
 }
+
```
Comment (qaz10102030) [inline client/lib/pages/block_goods_page.dart:14]: Does those empty line is generate by `dart format` command?

File: client/lib/pages/home.dart [hunk 2]
```diff
@@ -8,36 +11,66 @@ class Home extends StatelessWidget {
...
+           CheckoutColumn(),
+
         ],
       ),
-      drawer:const NavigationDrawer() ,
     );
   }
 }
+
+
```
Comment (qaz10102030) [inline client/lib/pages/home.dart:76]: Same as previous.

Comment (aniku777tw) [reply client/lib/pages/home.dart:76]: it was not generate by `dart format` .
It caused because I forgot to indent for each file.

**Review by qaz10102030 — APPROVED** [review]
LGTM

**Pull request merged**